samples/restrict_fw_rules_rdp_open.yaml

```diff
@@ -17,7 +17,7 @@
 apiVersion: constraints.gatekeeper.sh/v1alpha1
 kind: GCPRestrictedFirewallRulesConstraintV1
 metadata:
-  name: restrict-firewall-rule-ssh-open
+  name: restrict_firewall_rule_rdp_open
```

Comment:
Why switch to underscore? Kubernetes names can only use hyphens.

Reply:
Because majority of constraint names  (66 out of 75) in other samples are using underscore, and I want to make things consistent so that the report will look nicer. Are we going to change all to hyphens in order to be consistent with gatekeeper?

Reply:
Yes, I think we should be moving in the other direction (updating other constraint samples to use hyphens).

Reply:
ok!